### ISSUE TITLE
Announce when new cursor is added

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#109918]
Currently multi cursors are not accessible.
For starters we shuold announce whenever a new cursor is added. 

@alexdima let me know if you have a good code pointer

**Comment by alexdima** [comment]
@isidorn there are multiple ways to end up with multiple cursors. Which commands do you have in mind (ctrl+alt+up/down, column selection commands, ctrl+d, ctrl+shift+L)?

**Comment by isidorn** [comment]
@alexdima I hoped there was a common layer that all these commands used and that we could announce in that layer.
Since I think the experience should be the same, no matter the command.

But if we have to do it on a command layer - then all commands that are keyboard triggered make good sense.

**Comment by alexdima** [comment]
Here are the actions (their class names). They are living in `multicursor.ts`:
* `InsertCursorAbove`
* `InsertCursorBelow`
* `InsertCursorAtEndOfEachLineSelected`
* `InsertCursorAtEndOfLineSelected`
* `InsertCursorAtTopOfLineSelected`
* `AddSelectionToNextFindMatchAction`
* `AddSelectionToNextFindMatchAction`
* `MoveSelectionToNextFindMatchAction`
* `MoveSelectionToPreviousFindMatchAction`
* `SelectHighlightsAction`
* `CompatChangeAll`

**Comment by isidorn** [comment]
Thanks!
Assigning to this milestone so I take a look into this.

**Comment by isidorn** [comment]
I have pushed a change that we announce the cursors added after each of this commands in the format:
`Cursor added LINE:COLUMN`
If there are multiple cursors added we announce
`Cursor added LINE_1:COLUMN_1, LINE_2:COLUMN_2... LINE_N:COLUMN_N`

@alexdima if you have time you can take a look at the code changes.
I basically get the cursor state before the operation and after and announce the difference.

I will share this with the community and once they try it out I can hopefully push some additional improvements.

**Comment by isidorn** [comment]
Let's keep this issue open for additional feedback

**Comment by SamKacer** [comment]
Haven't tried this feature, but I really love this idea. I was thinking something like this should exist for a long time, since very often I would accidently add multi cursors without knowing and end up messing up a file. sometimes forcing me to revert a large size chunk of work and start over.

another idea I have, but this would probably be more appropriate as a extension than a built-in feature, wuld be to make some kind of inoffensive sound play whenever typing with multiple cursors. this would be analogous to NVDA beeping when typing with shift pressed when capslock is on. just a reminder that multi cursors are there.

**Comment by isidorn** [comment]
@SamKacer thanks a lot for providing feedback, if you get a chance try it out with VS Code insiders and let us know what you think about the changes we did https://code.visualstudio.com/insiders/

On the vscode side we try to not play any sounds, but to leave this up to the screen reader. So if there is some aria signal to make the screen reader produce a sound we could add it.
Also as you say an extension could tackle this.

**Comment by isidorn** [comment]
Closing this issue as the initial planned work is done.
Once the community has feedback we can tackle that in separate issues.
